tests/Tests/500__jupyterhub/test-filling-pvc.robot

```diff
@@ -0,0 +1,38 @@
+*** Settings ***
+Resource            ../../Resources/ODS.robot
+Resource            ../../Resources/Common.robot
+Resource            ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource            ../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+Library             DebugLibrary
+
+Suite Setup         Server Setup
+Suite Teardown      End Web Test
+
+
+*** Variables ***
+${PATH_TO_FILE}         ods-ci-notebooks-main/notebooks/200__monitor_and_manage/203__alerts/notebook-pvc-usage/fill-notebook-pvc-to-complete-100.ipynb
```

Comment:
This line seems too long (did you run robocop?). If you want to split it you'll find how to do it [here](https://github.com/red-hat-data-services/ods-ci/blob/f46af57ebd98e8db567570835e565cbd39b5f518/tests/Tests/200__monitor_and_manage/210__alerts/210__alerts.robot#L15)